### ISSUE TITLE
Low download speed in mypal

### ISSUE THREAD

**Issue by gusara47** [Feodor2/Mypal#144]
Download speed in mypal for some reason very low (like 150 kb\s) in comparison to other web browsers (firefox, chrome and serpent) 
Tested few versions on 2 computers (stable 29.3 and 68.13.8) results same, dont use any firewalls by the way. 
Whats the problem and what else can i check?

**Comment by Thyristor-Chopper** [comment]
is the file you downloading from the same site?

**Comment by gusara47** [comment]
Yes, i downloaded same file from the same site on diferent mypal versions and other browsers just for test. Download speed in mypal low in comparison with other browsers.
